hi-ml/src/health_ml/utils/wsi_utils.py

```diff
@@ -0,0 +1,19 @@
+from async_timeout import Any
+import torch
+import numpy as np
+from torch.utils.data.dataloader import default_collate
+from typing import List
+
+
+def list_data_collate(batch: List) -> Any:
+    """
+        Combine instances from a list of dicts into a single dict, by stacking them along first dim
+        [{'image' : 3xHxW}, {'image' : 3xHxW}, {'image' : 3xHxW}...] - > {'image' : Nx3xHxW}
+        followed by the default collate which will form a batch BxNx3xHxW
+    """
+
+    for i, item in enumerate(batch):
+        data = item[0]
```

Comment:
I'm a little confused, what are the datatypes here? the list comprehension creates a list of numpy arrays or tensors? can that be simplified?